samples/bluetooth/iso_time_sync/include/iso_time_sync.h

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef ISO_TIME_SYNC_H__
+#define ISO_TIME_SYNC_H__
+
+/** Definitions for the ISO time sync sample
+ *
+ * This file contains common definitions and API declarations
+ * used by the sample.
+ */
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <zephyr/bluetooth/iso.h>
+
+#define SDU_SIZE_BYTES 5 /* SDU = [btn_val, 4 byte sdu_counter]*/
+
+/** Start BIS transmitter demo
+ *
+ * @param retransmission_number    Requested retransmission number
+ * @param max_transport_latency_ms Requested maximum transport latency
+ */
+void bis_transmitter_start(uint8_t retransmission_number, uint16_t max_transport_latency_ms);
+
+/** Start BIS receiver demo
+ *
+ * @param bis_index_to_sync_to BIS index that it will sync to
```

Comment:
```suggestion
 * @param bis_index_to_sync_to BIS index that it will sync to.
```
In general, end all function and parameter descriptions with a full stop.